src/App.tsx

```diff
@@ -17,18 +53,38 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setOptionsValue={setOptionsValue}
+                setInputValue={setInputValue}
+                inputValue={inputValue}
+              />
             </div>
+            {loading && <Loader />}
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!loading && todos.length > 0 && (
+                <TodoList
+                  fillteredTodos={fillteredTodos}
```

Comment:
The prop 'fillteredTodos' passed to the TodoList component is misspelled. It should be 'filteredTodos'. Ensure consistency with the corrected variable name.